go.mod

```diff
@@ -4,11 +4,11 @@ go 1.17
 
 require (
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/opencontainers/runc v1.0.3
+	github.com/opencontainers/runc v1.1.0
```

Comment:
Should we update the spec too?

Reply:
can you elaborate a bit more?

Reply:
Sorry, I thought that the the spec may also be updated, but there is still only a [`v1.02` tag](https://github.com/opencontainers/runtime-spec/releases/tag/v1.0.2).